sample/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
 apply plugin: 'app.cash.paparazzi'
+apply plugin: 'jacoco'
```

Comment:
Is there an automated regression test that can be added?

Reply:
My thinking is applying the jacoco plugin is that regression test, since adding that one liner demonstrates the UnsupportedOperationException.  What else did you have in mind?

Reply:
added a plugin integration test, PTAL

Reply:
Nice!

Is it worth keeping this line in sample?

Reply:
(can't approve the PR)

Reply:
> Is it worth keeping this line in sample?

Let's leave it.  It still shows things are working by not failing as previously and we can build on this for CI checks in the future.

Reply:
@jrodbx this causes crazy spamming while running tests in :sample, repro: `gradlew :sample:test`

```
SEVERE: Unknown type (class [Z) in R class field: private static transient boolean[] app.cash.paparazzi.sample.R$array.$jacocoData
```

CI logs are 5x bigger than before. 200k to 1M, 2500 lines to 9000 lines.
https://github.com/cashapp/paparazzi/actions/runs/7151841385/job/19476541218#step:5:257

Related: https://github.com/cashapp/paparazzi/issues/732

Reply:
This is so annoying, even `spotlessApply` runs these tests...

Reply:
Is there a solution to this available? This happens in my project as well when I run unit tests...

Reply:
@markuskallander https://github.com/cashapp/paparazzi/issues/732#issuecomment-1490528929 is a workaround.

Reply:
@TWiStErRob the suggested workaround does not work. The fix would be to ignore synthetic fields as suggested here https://stackoverflow.com/a/52856548